openedx/features/course_experience/__init__.py

```diff
@@ -104,5 +104,13 @@ def course_home_url(course_key):
     Arguments:
         course_key (CourseKey): The course key for which the home url is being requested.
     """
+    from openedx.core.djangoapps.catalog.utils import get_course_data
+    course_key_str = '{}+{}'.format(course_key.org, course_key.course)
+    course_data = get_course_data(course_key_str, ['course_type'])
```

Comment:
I believe we should use a conditional that flags on both `course_type` and the `product_source.slug`. 

Eventually, the `product_source` will be (or should be) more granular than the "edX" vs. "2u" it is today, e.g. "edX", "getsmarter", "emeritus", etc. Even though the `course_type` is distinct to GetSmarter Exec Ed courses as is, I'd say it still remains a question whether the `course_type` for GetSmarter Exec Ed courses should be the same `course_type` as the pre-existing `executive-education` course type for OCM executive education courses instead of a distinct `executive-education-2u`, where the only differentiating factor between them should be the `product_source` moving forward. That said, that's not the reality we're in right now :/

FWIW, all the EMET APIs have been flagging Exec Ed courses solely based on the `product_source`, not the `course_type`; I believe we need the combination of both.

Reply:
done